sdk/cosmos/azure-cosmos/HISTORY.md

```diff
@@ -1,5 +1,9 @@
 # Release History
 
+## 4.0.0b6
+
+- Fixed bug in synchronized_request for media APIs. 
```

Comment:
We also need to add a list of the breaking changes, specifically the removal of the media configuration options and the MediaReadMode.

Reply:
@srinathnarayanan - after this last update to the release notes we should be able to merge :)